### ISSUE TITLE
Add / Create Getting started tutorial page(s) 

### ISSUE THREAD

**Issue by samccone** [marionettejs/marionettejs.com#75]
So everyone else has a pretty robust getting started / tutorial. I think we can do this with ease. Does anyone have thoughts or ideas on this?

thanks for the idea @jenius

**Comment by jdaudier** [comment]
So for the meetup i'm planning with Googlers and Luke (Ember), we're trying to come up with a CRUD app to demo. We're looking into this app as an idea: https://github.com/stefanpenner/ember-jobs

Maybe it's something we can use for our site once it's converted over. 

Question: Would we want the demo app to use Marionette modules or a 3rd party module loader like webpack or browserify?  I'm thinking the latter.

**Comment by jmeas** [comment]
I agree with @samccone:

> The idea is not to like throw browserify + every + grunt + plugin + ever at it, rather just a straight vanilla js + _.template app that people can just make.

I don't think that we should use Marionette.Module or any other module loader.

**Comment by jenius** [comment]
I agree but also think it's probably a good idea to also have an example of how it can be used with a commonjs loader since these are in very heavy use, and it's not exactly a straightforward process to do so.

I think a demo app would help, but what I was originally thinking was somewhat of a walkthrough of how to get things set up and what to use. Right now the docs are a collection of documented utilities, assuming that you fully understand backbone and the backbone pieces and properties that each utility is built on top of. For something just starting, this is too intimidating.

I feel like when I'm introducing marionette to new people I also have to explain why it's useful and what exactly it is. I typically say something along the lines of "backbone's view system is probably it's weakest portion, and requires a lot of manual effort to get your view actually on the page, then manage it once it's there, swap it out for other ones, etc. Marionette helps to make this easier through the use of regions, which are essentially areas of your page that you can move views in and out of without actually having to render or touch/place actual html. It also provides a nice collection of utilities for managing common view patterns in apps. For example, for lists of things, which come up all the time in apps, and for which marionette provides easier ways for lists to interact with their list items and vice versa."

Obviously there are plenty of other features, but in order to understand the basics and get started, I feel like regions and collection views are some of the strongest and most useful points -- the rest can be explored after. I guess what I'm getting after overall is that, at the moment, marionette's docs present it as a straight up grab bag of classes with no real explanation of the context they fit into, how to fit them together into something that works, or where to get started with it. For an experienced backbone or marionette dev, this is fine, as you are familiar with the classes and just need some info so you know where to go. But for someone that's just starting out, this is massively confusing and usually drives them away unless they have heavy guidance and mentorship.
